go.mod

```diff
@@ -37,6 +37,7 @@ require (
 	github.com/golang-jwt/jwt/v5 v5.2.2
 	github.com/google/go-containerregistry v0.20.3
 	github.com/google/go-github/v56 v56.0.0
+	github.com/google/go-github/v71 v71.0.0
```

Comment:
Looks like we potentially need to move away from `github.com/google/go-github/v56`?

Reply:
🤦‍♂️ That's my b. Forgot we were already using an older version of this module somewhere.

Agree we should remove 56 and just use 71 across the board.

Reply:
I think it's no one's fault really, it appears like the way the module does versioning isn't picked up by Dependabot (or something like `go-mod-upgrade`) because every minor version is _also_ a new versioned Go module (i.e. `/v56 => v56.0.0`, `v71 => `v71.0.0`, etc.).

Reply:
To solve the Dependabot issue, we likely need to create a specific rule that allows "major" updates for this specific dependency.